### PR TITLE
Chris/Follow Tab Update

### PR DESCRIPTION
**Pull request by ckdesir**
## Overview

Addresses some of the issues from GitHub via Lesley.

## Changes Made

- [x] Adds a SwipeRefreshLayout to refresh following publications
- [x] Switch to ViewPager2
- [x] Prevents swiping back to OnboardingFrag

## Next Steps (delete if not applicable)

- [x] Deal with the rest of the issues presented; feedback from initial launch when discussed next pod meeting
- [x] A SwipeRefreshLayout for the main Home screen

## Related PRs or Issues
#20 
#21 (partially)

## Screenshots 

<details>

  <summary>Follow Tab Update</summary>

https://user-images.githubusercontent.com/60499584/111107682-b9abd200-852d-11eb-9909-2252d8dfff63.mp4

</details>

### CODE REVIEW

**Comment by ckdesir** [comment]
apologize for this being a lot, couldn't stop changing things

File: app/src/main/java/com/cornellappdev/volume/views/WebviewTop.kt [hunk 1]
```diff
@@ -34,7 +34,7 @@ class WebviewTop @JvmOverloads constructor(
     }
 
     fun setName(article: Article) {
-        name.text = if (article.title!!.length < 40) article.title else article.title.substring(0, 41) + "..."
```
Comment (ckdesir) [inline app/src/main/java/com/cornellappdev/volume/views/WebviewTop.kt:37]: I think this was causing the crashes Lesley was experiencing (as it was crashing for me clicking on articles w/ exactly 41 characters, will be more explicit on some crash screen however

**Comment by tedim52** [comment]
Nice and quick PR Chris :) Small comment, write your commits in imperative mood. https://chris.beams.io/posts/git-commit/#imperative

**Review by kevinsun-dev — APPROVED** [review]
Couldn't find much issue with anything. LGTM

**Pull request merged**